src/satellitePropagator/io/FileOutput.h

```diff
@@ -282,7 +282,7 @@ void FileOutput<Container>::writeDebrisCSV(double t)
 }
 
 template <class Container>
-void FileOutput<Container>::writeDebrisTXT(double t)
+void FileOutput<Container>::writeDebrisTXT(double)
```

Comment:
was this intentional? What does it actually do 🤔

Reply:
Yes this was intentional. By removing the name of the argument it becomes inaccessible, we signal that we will not use that parameter, and the warning that it is not used is removed. At the same time, the signature of the function stays intact.